flow360/component/simulation/models/solver_numerics.py

```diff
@@ -328,9 +421,6 @@ class HeatEquationSolver(GenericSolverSettings):
         Frequency at which to solve the heat equation in conjugate heat transfer simulations
 
 
```

Comment:
Same here. Can you remove all the 
```
Paramters
---------
```

Reply:
Removed all parameters sections in the three modules documented in this PR.